src/Flex.php

```diff
@@ -1010,14 +965,13 @@ public static function getSubscribedEvents(): array
             ScriptEvents::POST_INSTALL_CMD => 'install',
             ScriptEvents::PRE_UPDATE_CMD => 'configureInstaller',
             ScriptEvents::POST_UPDATE_CMD => 'update',
-            ScriptEvents::POST_AUTOLOAD_DUMP => 'updateAutoloadFile',
-            'auto-scripts' => 'executeAutoScripts',
+            ScriptEvents::POST_AUTOLOAD_DUMP => [['auto-scripts'], ['executeAutoScripts']],
```

Comment:
This change looks wrong to me. There is no `auto-scripts` method. And we were not executing auto scripts on `POST_AUTOLOAD_DUMP`

Reply:
```auto-scripts``` should now map to ```executeAutoScripts``` method and removed ``` ScriptEvents::POST_AUTOLOAD_DUMP``` as this was not there prior to the change in #576. If there are any issues with this, let me know!